arwen/contexts/output.go

```diff
@@ -345,11 +345,12 @@ func mergeVMOutputs(leftOutput *vmcommon.VMOutput, rightOutput *vmcommon.VMOutpu
 	if leftOutput.OutputAccounts == nil {
 		leftOutput.OutputAccounts = make(map[string]*vmcommon.OutputAccount)
 	}
-	for _, rightAccount := range rightOutput.OutputAccounts {
-		leftAccount, ok := leftOutput.OutputAccounts[string(rightAccount.Address)]
+
+	for address, rightAccount := range rightOutput.OutputAccounts {
```

Comment:
please move it back :) it is much safer.

Reply:
Reverted.

Reply:
See above.